submarine-workbench/submarine-server/pom.xml

```diff
@@ -181,6 +181,12 @@
       <artifactId>slf4j-log4j12</artifactId>
       <version>${slf4j.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.mybatis.generator</groupId>
+      <artifactId>mybatis-generator-core</artifactId>
+      <version>1.3.7</version>
+    </dependency>
```

Comment:
All the sub module's dependency should put out at the parent pom file.

Reply:
@jiwq , Do not understand what you mean?
If the submodule's dependencies are placed in the parent module, then other submodules will also be included this dependencies.